components/script/dom/create.rs

```diff
@@ -276,7 +276,8 @@ fn create_html_element(name: QualName,
 pub fn create_element(name: QualName,
                       prefix: Option<Prefix>,
                       document: &Document,
-                      creator: ElementCreator)
+                      creator: ElementCreator,
+                      line_number: u64)
```

Comment:
Given how many places end up creating elements and passing fake line number values, I propose we modify the ElementCreator enum such that the `ParserCreated` variant takes a line number argument instead. This will mean that we don't need an extra argument here.